fixtures/packaging/babel-standalone.html

```diff
@@ -1,14 +0,0 @@
-<html>
```

Comment:
Oh I see - it was nested under 'packaging' and didn't seem like that belonged. I can put it back though.